.github/workflows/branch.yml

```diff
@@ -13,7 +13,7 @@ env:
   BUCKET_WEB: reactive-trader-web-builds
   BUCKET_OPENFIN: reactive-trader-openfin-builds
   BUCKET_FINSEMBLE: reactive-trader-finsemble-builds
-  VITE_HYDRA_URL: wss://trading-web-gateway-rt.demo.hydra.weareadaptive.com
+  VITE_HYDRA_URL: wss://trading-web-gateway-rt-dev.demo.hydra.weareadaptive.com
```

Comment:
ah, interesting .. so we have both now, I guess

Reply:
Yep, David sorted our reactive-deployments bits yesterday too.
I don't think we actually need/use this variable here in the application, but updated and leaving in for now until I'm more confident on what's happening.